app/src/server/db/HomeService.ts

```diff
@@ -0,0 +1,24 @@
+import { PrismaClient } from "@prisma/client";
+
+export const getHomesByUserId = async (id: string, db: PrismaClient): Promise<string[]> => {
```

Comment:
Nice, didn't know you could do this that way. Good work, makes server side alot easier.